pom.xml

```diff
@@ -107,78 +101,69 @@
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
 		<dependency>
-		  <groupId>io.fabric8</groupId>
-		  <artifactId>kubernetes-client</artifactId>
-		  <version>5.3.1</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>kubernetes-client</artifactId>
+			<version>5.3.2</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>tekton-client</artifactId>
-		    <version>5.5.0</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>tekton-client</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>tekton-model-v1beta1</artifactId>
-		    <version>5.5.0</version>	      
+			<groupId>io.fabric8</groupId>
+			<artifactId>tekton-model-v1beta1</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>tekton-model-triggers</artifactId>
-		    <version>5.5.0</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>tekton-model-triggers</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>knative-model</artifactId>
-		    <version>5.5.0</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>knative-model</artifactId>
+			<version>5.5.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-devtools</artifactId>
 			<optional>true</optional>
 		</dependency>
 		<dependency>
-			<groupId>io.springfox</groupId>
-			<artifactId>springfox-swagger2</artifactId>
-			<version>2.9.2</version>
-			<exclusions>
-				<exclusion>
-					<groupId>com.google.guava</groupId>
-					<artifactId>guava</artifactId>
-				</exclusion>
-			</exclusions>	
-		</dependency>
-		<dependency>
-		    <groupId>com.google.guava</groupId>
-		    <artifactId>guava</artifactId>
-		    <version>28.0-jre</version>
-		</dependency>
```

Comment:
What was guava used for?

Reply:
It was used for springfox. No longer needed with migration to springdoc